edge-server/anthos/generate-script.sh

```diff
@@ -264,6 +264,8 @@ sed -i 's/${ANTHOS_SERVICE_ACCOUNT_KEY_PATH}/\/var\/lib\/viai\/setup\/edge-serve
 
 echo "Updating env.sh..."
 # shellcheck disable=SC2016
+sed -i 's/${ANTHOS_VERSION}/'"${ANTHOS_VERSION}"'/g' "$OUTPUT_FOLDER/edge-server/env.sh"
```

Comment:
Can you please add a quick test that this happened? You already run `scripts/1-generate-edge-server-assets.sh` in `scripts/test-shell-scripts.sh`. After running that, you should have the `$OUTPUT_FOLDER/edge-server/env.sh` script with the correct values. At that point, you can source that `env.sh`, source `common.sh` as well and compare if the two `ANTHOS_VERSION` variables contain the same value. Maybe you'll need to store their values in new, temporary variables because they have the same name both in `env.sh` and `common.sh`.

Reply:
add a `test-anthos-version.sh` to compare version in common.sh and in generated .sh file.
PTAL